{{cookiecutter.repo_name}}/requirements/base.txt

```diff
@@ -34,4 +34,8 @@ django-autoslug==1.8.0
 # Time zones support
 pytz==2015.4
 
+# Redis support
```

Comment:
I put it here because I did not want to have requirements defined in two places. Previously the only requirement at `/requirements.txt` had to do with memcache, and the rest was just a proxy to the actual requirements (as per the comment in the file).